tests/blockchain/eth/contracts/contracts/WorkLockTestSet.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.5.3;
```

Comment:
Is this still the pragma we want to specify?

Reply:
it works more like restriction >= 0.5.3, < 0.6.0, means that you can use 0.5.3 if you want, so it's still ok, maybe I'll change it to higher version in one of the next PR

Reply:
Isn't it the version we're using in all our contracts?

Reply:
We have 0.5.9 in installation script, main reason for 0.5.3 in contracts is that no changes in syntax after 0.5.3 (or maybe even earlier). But I'll try different versions from optimization point of view in next PR